### PR TITLE
grids problem implementation

### PR DESCRIPTION
**Pull request by danielpinheiros**

### CODE REVIEW

File: glTFRevitExport/Extensions/RevitAPIExtensions.cs [hunk 1]
```diff
@@ -137,5 +137,24 @@ public static bool IsBIC(this Category c, BuiltInCategory bic)
...
+        {
+            if (bbox == null || curve == null)
+            {
+                return false;
+            }
+
+            var startPoint = curve.Evaluate(0, true);
+            var endPoint = curve.Evaluate(1, true);
+
+            var b1 = new Vector3D((float)bbox.Min.X, (float)bbox.Min.Y, (float)bbox.Min.Z);
```
Comment (gastonagusti) [inline glTFRevitExport/Extensions/RevitAPIExtensions.cs:151]: maybe we could create a method that returns a Vector3D from the bounding box. 
Vector3D CreateVectorFromBBox(BoundingBox bbox)

File: glTFRevitExport/Utils/GeometryUtils.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            if ((fDst1 * fDst2) >= 0.0f) return false;
+            if (fDst1 == fDst2) return false;
+            Hit = P1 + (P2 - P1) * (-fDst1 / (fDst2 - fDst1));
+            return true;
+        }
+
+        private static bool InBox(Vector3D Hit, Vector3D B1, Vector3D B2, int Axis)
+        {
+            if (Axis == 1 && Hit.Z > B1.Z && Hit.Z < B2.Z && Hit.Y > B1.Y && Hit.Y < B2.Y) return true;
```
Comment (gastonagusti) [inline glTFRevitExport/Utils/GeometryUtils.cs:44]: try to put the hardcodes like 1, 2, 3 in constant with representatives names

**Review by gastonagusti — APPROVED** [review]

File: glTFRevitExport/Extensions/RevitAPIExtensions.cs (hunk 1 above)
Comment (danielpinheiros) [reply glTFRevitExport/Extensions/RevitAPIExtensions.cs:151]: perfect! Done!

File: glTFRevitExport/Utils/GeometryUtils.cs (hunk 2 above)
Comment (danielpinheiros) [reply glTFRevitExport/Utils/GeometryUtils.cs:44]: Great point! I've implemented it!

**Pull request merged**